Makefile

```diff
@@ -0,0 +1,297 @@
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+# Image URL to use all building/pushing image targets
+IMG ?= numaplane-controller
+VERSION ?= latest
+# BASE_VERSION will be used during release process to bump up versions
+BASE_VERSION := latest
+# Default cluster name where numaplane get deployed, update it as needed.
+CLUSTER_NAME ?= staging-usw2-k8s
+IMAGE_NAMESPACE ?= quay.io/numaproj
```

Comment:
cool, so now we're back to quay.io/numaproj to distinguish our previous images at quay.io/numaio from the new ones